bootstrap/pkg/kfapp/coordinator/coordinator.go

```diff
@@ -385,29 +337,30 @@ func LoadKfAppCfgFile(cfgfile string) (kftypesv3.KfApp, error) {
 	// Set some defaults
 	// TODO(jlewi): This code doesn't belong here. It should probably be called from inside KfApp; e.g. from
 	// KfApp.generate. We should do all initialization of defaults as part of the reconcile loop in one function.
-	if c.KfDef.Spec.PackageManager == "" {
-		c.KfDef.Spec.PackageManager = kftypesv3.KUSTOMIZE
-	}
+	// if c.KfDef.Spec.PackageManager == "" {
```

Comment:
Why did you comment this out?

Reply:
we don't need packagemanager field right?
It's always kustomize.

Reply:
+1